### PR TITLE
cl workers -- worker visibility

### PR DESCRIPTION
**Pull request by fabeschan**
To do:

- [x] Test root user only restriction

### CODE REVIEW

File: codalab/lib/bundle_cli.py [hunk 1]
```diff
@@ -60,7 +60,8 @@
 from codalab.lib.cli_util import (
     nested_dict_get,
     parse_target_spec,
-    desugar_command
+    desugar_command,
+    ReadableDateTimeDelta,
```
Comment (percyliang) [inline codalab/lib/bundle_cli.py]: Is this used?

File: codalab/client/json_api_client.py [hunk 2]
```diff
@@ -678,3 +678,10 @@ def update_worksheet_raw(self, worksheet_id, lines):
             path='/worksheets/%s/raw' % worksheet_id,
             headers={'Content-Type': 'text/plain'},
             data='\n'.join(lines))
+
+    @wrap_exception('Unable to fetch worker information')
+    def get_worker_info(self):
```
Comment (percyliang) [inline codalab/client/json_api_client.py]: rename to `get_workers_info` (plural); also applies to stuff in `bundle_cli.py`.

File: codalab/rest/workers.py [hunk 3]
```diff
@@ -190,6 +191,23 @@ def finalize_bundle(worker_id, uuid):
                                 request.json['exitcode'],
                                 request.json['failure_message'])
 
+@get('/workers/info', name='worker_info', apply=AuthenticatedPlugin())
+def worker_info():
+    '''
+    if request.user.user_id != local.model.root_user_id:
```
Comment (percyliang) [inline codalab/rest/workers.py]: Why is this commented out?

File: codalab/rest/workers.py [hunk 4]
```diff
@@ -190,6 +191,23 @@ def finalize_bundle(worker_id, uuid):
                                 request.json['exitcode'],
                                 request.json['failure_message'])
 
+@get('/workers/info', name='worker_info', apply=AuthenticatedPlugin())
+def worker_info():
```
Comment (percyliang) [inline codalab/rest/workers.py]: `workers_info`

**Review by percyliang — CHANGES_REQUESTED** [review]
This will be great to have!  We can follow up with mem,%cpu,disk reporting in a separate PR.

**Comment by fabeschan** [comment]
Fixed nits

**Pull request merged**